docs/src/solutions.md

```diff
@@ -116,6 +117,84 @@ end
 # TODO: How to accurately measure the solve time.
 ```
 
+## Sensitivity analysis
+
+Given an LP problem and an optimal solution corresponding to a basis, we can
+question how much an objective coefficient or standard form rhs coefficient
+(c.f., [`standard_form_rhs`](@ref)) can change without violating primal or dual
+feasibility of the basic solution.
+
```

Comment:
Please add a note that this form of sensitivity analysis appears often in textbooks but has limited use in practice because optimal bases are often degenerate.

Reply:
(If you object to this characterization, I'd be interested to hear the argument.)

Reply:
True, it depends quite a lot on the instance if these ranges are practically useful, I added a paragraph discussing briefly their limitations.